### ISSUE TITLE
Checkbox border in "Getting started" dialog has insufficient contrast against its adjacent color

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#857]
The following AI-windows mode(s) have this issue:
- Regular mode
- High Contrast White mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![4 1](https://user-images.githubusercontent.com/22068434/88343153-9958aa00-ccf5-11ea-8f33-4a35ffc26acb.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
This is fixed by #943.

@iamrafan, this fix will be in release 1.1.1370.01 (will release October 2nd). Please regress and close once the build is out.

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1380.01
